File: src/programmers/helpers/ProtobufWire.ts
export const enum ProtobufWire {
  /**
   * - integers
   * - bool
   * - enum
   */
  VARIANT = 0,

  /**
   * - fixed64
   * - sfixed64
   * - double
   */
  I64 = 1,

  /**
   * - string
   * - bytes
   * - mebedded messages
   * - packed repeated fields
   */
  LEN = 2,

  START_GROUP = 3,

  END_GROUP = 4,

  /**
   * - fixed
   * - sfixed32
   * - float
   */
  I32 = 5,
}

File: packages/typia/native/core/programmers/helpers/ProtobufWire.go
package helpers

type ProtobufWire int

const (
  VARIANT ProtobufWire = 0
  I64     ProtobufWire = 1
  LEN     ProtobufWire = 2

  START_GROUP ProtobufWire = 3
  END_GROUP   ProtobufWire = 4

  I32 ProtobufWire = 5
)
